scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+            if not csv_files:
+                return prefecture, []
+            
+            for csv_file in csv_files:
+                with pref_zf.open(csv_file) as f:
+                    content_bytes = f.read()
+                    
+                    for encoding in ['shift_jis', 'cp932', 'utf-8']:
+                        try:
+                            content = content_bytes.decode(encoding)
+                            break
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)